src/containers/LoginForm/LoginForm.js

```diff
@@ -1,20 +1,82 @@
-import React from 'react'
+import React, { useState } from "react";
+import { Button, TextField, Container } from "@material-ui/core";
 
-import {
-  Button,
-  TextField,
-} from '@material-ui/core'
+import "./style.css";
 
-import './style.css'
+const LoginForm = ({ toSend }) => {
+  const [password, setPassword] = useState("");
+  const [user, setUser] = useState("");
+  const [errors, setErrors] = useState({
+    validator: { buttonEnabled: false, isvalid: true, text: "" },
+  });
 
-const LoginForm = () => (
-  <form className="loginForm" noValidate autoComplete="off">
-    <div className="loginForm--inputs">
-      <TextField id="standard-basic" label="Usuário" />
-      <TextField id="standard-basic" label="Senha" />
-    </div>
-    <Button variant="contained">Entrar</Button>
-  </form>
-)
+  function passwordValidation(password) {
+    if (password.length < 8) {
+      return {
+        buttonEnabled: false,
+        validate: false,
+        text: "A senha deve ter pelo menos 8 dígitos",
+      };
+    } else {
+      return { buttonEnabled: true, valido: true, text: "" };
+    }
+  }
```

Comment:
No caso, não precisa desse `else` aqui, porque se a condicional não passar no `if`, ele já passaria pro return.
```suggestion
    } 
      return { buttonEnabled: true, valido: true, text: "" };
    }
  }
```